### PR TITLE
Added targetHost to query and set targeted mesh with ArcRotateCamera

### PR DESCRIPTION
**Pull request by CedricGuillemet**
followup https://forum.babylonjs.com/t/getting-the-mesh-set-by-cameras-settarget/24152/4

I set the property as serializable. Is it ok?

### CODE REVIEW

File: src/Cameras/arcRotateCamera.ts [hunk 1]
```diff
@@ -66,6 +67,17 @@ export class ArcRotateCamera extends TargetCamera {
         this.setTarget(value);
     }
 
+    /**
+     * Defines the target mesh of the camera.
+     * The camera looks towards it form the radius distance.
+     */
+    public get targetHost(): AbstractMesh {
```
Comment (RaananW) [inline src/Cameras/arcRotateCamera.ts:74]: Why is this not `Nullable<AbstractMesh>`?

File: src/Cameras/arcRotateCamera.ts [hunk 2]
```diff
@@ -66,6 +67,17 @@ export class ArcRotateCamera extends TargetCamera {
         this.setTarget(value);
     }
 
+    /**
+     * Defines the target mesh of the camera.
+     * The camera looks towards it form the radius distance.
```
Comment (Popov72) [inline src/Cameras/arcRotateCamera.ts:72]: nit: "from"

**Review by Popov72 — APPROVED** [review]

**Comment by RaananW** [comment]
any update on this one?

File: src/Cameras/arcRotateCamera.ts (hunk 1 above)
Comment (CedricGuillemet) [reply src/Cameras/arcRotateCamera.ts]: fixed

File: src/Cameras/arcRotateCamera.ts (hunk 2 above)
Comment (sebavan) [reply src/Cameras/arcRotateCamera.ts:72]:
```suggestion
     * The camera looks towards it from the radius distance.
```

**Pull request merged**